sites/svelte.dev/src/routes/search/+page.svelte

```diff
@@ -11,7 +11,7 @@
 <main>
 	<h1>Search</h1>
 	<form>
-		<input name="q" value={data.query} placeholder="Search" spellcheck="false" />
+		<input name="q" value={data.query} type="search" placeholder="Search" spellcheck="false" />
```

Comment:
This input has a larger accessibility issue - it's missing a label. Can you add `aria-label="Search"` to the field as well?

Reply:
> This input has a larger accessibility issue - it's missing a label. Can you add `aria-label="Search"` to the field as well?

Done 🙌🏽